shell/platform/windows/egl/egl.cc

```diff
@@ -0,0 +1,67 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/egl/egl.h"
+
+#include <EGL/egl.h>
+
+#include "flutter/fml/logging.h"
+
+namespace flutter {
+namespace egl {
+
+namespace {
+
+const char* EGLErrorToString(EGLint error) {
+  switch (error) {
+    case EGL_SUCCESS:
+      return "Success";
+    case EGL_NOT_INITIALIZED:
+      return "Not Initialized";
+    case EGL_BAD_ACCESS:
+      return "Bad Access";
+    case EGL_BAD_ALLOC:
+      return "Bad Alloc";
+    case EGL_BAD_ATTRIBUTE:
+      return "Bad Attribute";
+    case EGL_BAD_CONTEXT:
+      return "Bad Context";
+    case EGL_BAD_CONFIG:
+      return "Bad Config";
+    case EGL_BAD_CURRENT_SURFACE:
+      return "Bad Current Surface";
+    case EGL_BAD_DISPLAY:
+      return "Bad Display";
+    case EGL_BAD_SURFACE:
+      return "Bad Surface";
+    case EGL_BAD_MATCH:
+      return "Bad Match";
+    case EGL_BAD_PARAMETER:
+      return "Bad Parameter";
+    case EGL_BAD_NATIVE_PIXMAP:
+      return "Bad Native Pixmap";
+    case EGL_BAD_NATIVE_WINDOW:
+      return "Bad Native Window";
+    case EGL_CONTEXT_LOST:
+      return "Context Lost";
+  }
+  return "Unknown";
```

Comment:
I assume this is all of them? If so, and if we're checking that we have 100% coverage with static analysis (we should　be on Linux but not sure if we are on Windows), then consider adding an `FML_DCHECK(false)` to blow up here.

Reply:
Ah I see it looks like it's an `EGLint` not an enum :/

Reply:
Yes this is supposed to be all of them. I added an `FML_UNREACHABLE`. I think it's a good trade-off: the potential short-term impact (a debug-only crash) nets better error messages in the long-term.